guide/case-studies/cloud-backup.md

```diff
@@ -43,85 +20,172 @@ https://www.figma.com/community/file/968416729557947210
+   retina = "/assets/images/guide/case-studies/cloud-backup/daily-spending-wallet-onboarding-begin@2x.png"
+   width = 250
+   height = 541
+   caption = "Offer clear options to create and restore wallets."
+   alt-text = "Cover screen user interface"
+   layout = "shadow"
+%}
+
+{% include image.html
+   image = "/assets/images/guide/case-studies/cloud-backup/daily-spending-wallet-onboarding-benefits.png"
+   retina = "/assets/images/guide/case-studies/cloud-backup/daily-spending-wallet-onboarding-benefits@2x.png"
+   width = 250
+   height = 541
+   caption = "Talk about ownership"
+   alt-text = "Cover screen user interface"
+   layout = "shadow"
+%}
 
-{% capture imageURL %}{{ page.image_base }}{{ item.file }}.png{% endcapture %}
-{% capture imageURLRetina %}{{ page.image_base }}{{ item.file }}@2x.png{% endcapture %}
+{% include image.html
+   image = "/assets/images/guide/case-studies/cloud-backup/daily-spending-wallet-onboarding-cloud.png"
+   retina = "/assets/images/guide/case-studies/cloud-backup/daily-spending-wallet-onboarding-cloud@2x.png"
+   width = 250
+   height = 541
+   caption = "Talk about recovery"
+   alt-text = "Cover screen user interface"
+   layout = "shadow"
+%}
 
-{% include picture.html
-   image = imageURL
-   retina = imageURLRetina
-   alt-text = item.alt
-   caption = item.caption
+{% include image.html
+   image = "/assets/images/guide/case-studies/cloud-backup/daily-spending-wallet-onboarding-security.png"
+   retina = "/assets/images/guide/case-studies/cloud-backup/daily-spending-wallet-onboarding-security@2x.png"
    width = 250
    height = 541
+   caption = "Talk about security"
+   alt-text = "Cover screen user interface"
+   layout = "shadow"
+%}
+
+{% include image.html
+   image = "/assets/images/guide/case-studies/cloud-backup/daily-spending-wallet-onboarding-home.png"
+   retina = "/assets/images/guide/case-studies/cloud-backup/daily-spending-wallet-onboarding-home@2x.png"
+   width = 250
+   height = 541
+   caption = "As key creation and backup is automated, users can immediately be navigated to the home screen."
+   alt-text = "Cover screen user interface"
    layout = "shadow"
 %}
-{% endfor %}
 
 </div>
 
-### Importing a wallet
+### Recovering a wallet
 
-The import sequence guides the user through entering the recovery phrase, which is then also encrypted and saved to cloud storage.
+The recovery flow guides the user through restoring their wallet on a new device. For example, the user creates their wallet with their smartphone, then upgrades to a new phone, and wants to "transfer" the wallet to their new phone.
 
 <div class="image-slide-gallery">
 
-{% for item in page.importImages %}
+{% include picture.html
+   image = "/assets/images/guide/case-studies/cloud-backup/daily-spending-wallet-restore-menu.png"
+   retina = "/assets/images/guide/case-studies/cloud-backup/daily-spending-wallet-restore-menu@2x.png"
+   width = 250
+   height = 600
```

Comment:
Height for these images should also be 541 to match the proportions of the image files.

The image files should ideally also be 250x541, they are currently 375*812.

Reply:
Fixed